packages/components/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { User } from '@codesandbox/common/lib/types';
+import styled from 'styled-components';
+import { Element } from '../Element';
+import { Text } from '../Text';
+
+interface IAvatarProps {
+  user: User;
+}
+
+export const AvatarImage = styled.img(
+  css({
+    width: 8,
+    height: 8,
+    borderRadius: 'small',
+    border: '1px solid',
+    borderColor: 'sideBar.border',
+    backgroundColor: 'sideBar.border', // fallback for loading image
+  })
+);
+
+export const Pro = styled(Text).attrs({ size: 1, weight: 'bold' })(
+  css({
+    backgroundColor: 'blues.700',
+    color: 'white',
+    borderRadius: 'small',
+    paddingX: 2,
+    paddingY: '2px', // no 2 in spaces
+    border: '1px solid',
+    borderColor: 'sideBar.border',
```

Comment:
To make sure we handle this across themes, please add a new key `avatar.border` and add it to polyfills: https://github.com/codesandbox/codesandbox-client/blob/master/packages/components/src/utils/polyfill-theme.ts#L15

The colors I'm setting for now is what makes sense for codesandbox black and then we will revisit them to switch to a layer method that works for all themes